controllers/apps/configuration/configconstraint_controller.go

```diff
@@ -48,7 +48,9 @@ type ConfigConstraintReconciler struct {
 // +kubebuilder:rbac:groups=apps.kubeblocks.io,resources=configconstraints/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=apps.kubeblocks.io,resources=configconstraints/finalizers,verbs=update
 
-// +kubebuilder:rbac:groups=apiextensions.k8s.io,resources=customresourcedefinitions,verbs=get;update;patch
+// +kubebuilder:rbac:groups=apiextensions.k8s.io,resources=customresourcedefinitions,verbs=get;create;update;patch
```

Comment:
why controller needs the access permission?

Reply:
The controller doesn't pay attention to this comment, as it can be placed anywhere. Its primary purpose is to enable the controller-tools to automatically generate RBAC information.